backend/ctgov/models.py

```diff
@@ -1065,3 +1066,241 @@ class Y2018MeshTerms(models.Model):
     class Meta:
         managed = False
         db_table = "y2018_mesh_terms"
+
+
+class BasicSearch(models.Model):
+    status = models.TextField(blank=True, null=True)
+    brief_title = models.TextField(blank=True, null=True)
+    official_title = models.TextField(blank=True, null=True)
+    nct_id = models.TextField(blank=True, null=True)
```

Comment:
Since we are using nct_id as the pseudo primary key, what is the advantage to having this nullable?

Reply:
These parameters are useful only when we insert or update. Since we are just fetching from existing database, I kept Django generated parameters as is.

Reply:
Resolving, but may request updates in the future depending only on if the ETL process uses these models.